--- /usr/local/src/rstudio-rstudio-ca19c52/src/cpp/session/modules/SessionWorkbench.cpp	2013-05-11 02:43:04.000000000 -0700
+++ ./SessionWorkbench.cpp	2013-07-14 11:44:16.985235490 -0700
@@ -146,6 +146,7 @@
       terminalPaths.push_back(FilePath("/usr/bin/gnome-terminal"));
       terminalPaths.push_back(FilePath("/usr/bin/konsole"));
       terminalPaths.push_back(FilePath("/usr/bin/xfce4-terminal"));
+      terminalPaths.push_back(FilePath("/usr/bin/mate-terminal"));
       terminalPaths.push_back(FilePath("/usr/bin/xterm"));
 
       BOOST_FOREACH(const FilePath& terminalPath, terminalPaths)
